{{cookiecutter.pipeline_slug}}/docs/installation.md

```diff
@@ -41,3 +41,9 @@ unzip master.zip -d /my-pipelines/
 cd /my_data/
 nextflow run /my-pipelines/{{ cookiecutter.pipeline_slug }}-master
 ```
+
+If you are running the pipeline on an offline resource (e.g. a private cluster without internet connection available upon pipeline running), you may specify an environment variable as above to avoid download failure messages when starting the pipeline. Just add this to your environment or profile as above:
```

Comment:
This kind of repeats the previous text a little, then refers to "as above" when the example is below... 😉 

How about:

> To stop nextflow from looking for updates online, you can tell it to run in offline mode by specifying the following environment variable in your `~/.bashrc` file: